### PR TITLE
Jd 2023 01 rename accordion

### PR DESCRIPTION
**Pull request by goapunk**

### CODE REVIEW

File: webpack.common.js [hunk 1]
```diff
@@ -86,6 +86,10 @@ module.exports = {
     swiper_phases: {
       import: './meinberlin/assets/js/swiper_phases.js'
     },
+    init_dashboard_accordion: {
```
Comment (goapunk) [inline webpack.common.js:89]: separate entry point as done in aplus

File: meinberlin/templates/a4dashboard/base_dashboard_project.html [hunk 2]
```diff
@@ -61,4 +61,5 @@
 
 {% block extra_js %}
     <script src="{% static 'unload_warning.js' %}"></script>
+    <script src="{% static 'init_dashboard_accordion.js' %}"></script>
```
Comment (goapunk) [inline meinberlin/templates/a4dashboard/base_dashboard_project.html:64]: this should cover all the places it's used I think?

**Comment by goapunk** [comment]
this will break the existing cookie but that's ok I guess?

**Review by phillimorland — APPROVED** [review]
very nice!

**Pull request merged**